.github/workflows/release.yaml

```diff
@@ -202,7 +220,9 @@ jobs:
       - uses: richb-hanover/cargo@v1.1.0
         with:
           command: release
-          args: publish --no-confirm -x
+          args:
+            publish --no-confirm ${{ github.event.inputs.dry-run == 'true' && ''
+            || '--execute' }}
```

Comment:
```suggestion
          # In a dry-run, we only attempt to publish `prql-ast`, because of
          # https://github.com/crate-ci/cargo-release/issues/691
          run:
            cargo release publish --no-confirm
            ${{ github.event.inputs.dry-run == 'true' 
            && '--execute' 
            || 'cargo release publish --dry-run -p prql-ast' }}
```